### DISCUSSION TITLE
Wiki edits

### DISCUSSION THREAD

**Discussion by asoftbird** [TriliumNext/Trilium#3104] [General]
Hi,

I'd like to update some pages on the wiki, but Git still hasn't made a feature to make pull requests for wiki pages, so I'm kinda stumped on how to submit my edits for review without it being a ton of work for the reviewer to ensure things end up overwriting the right pages..

I've tried a bunch of search engine results which didn't quite help and will probably cause more problems than it solves. 

How can I help out editing the wiki?

**Comment by zadam** [comment]
Hi, I added you as a collaborator so you should be able to edit wiki.

Wiki used to be open for editing until recently when there were some attacks on it.
